packages/roosterjs-editor-api/test/format/setOrderedListNumberingTest.ts

```diff
@@ -0,0 +1,81 @@
+import * as TestHelper from '../TestHelper';
+import { IEditor, PositionType } from 'roosterjs-editor-types';
+import { safeInstanceOf } from 'roosterjs-editor-dom/lib';
+import { setOrderedListNumbering } from 'roosterjs-editor-api';
+
+describe('setOrderedListNumberingTest()', () => {
```

Comment:
Suggest move most of the test code to test the split and writeBack function for VList (in VListTest), and for this test, we just need to make sure we call VList with correct parameters.